### PR TITLE
Forms: fix field bugs

### LINKED ISSUE #247: Forms - field bugs
# Location form
### opening_exceptions
We have marked as required all 4 fields (`title`, `is_open`, `start_date`, `end_date`), but we only add the asterisk to 2 of them (`start_date`, `end_date`).
https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/locations/schemas/locations/location-v1.0.0.json#L112

# Eitem form
### urls
The field `login_required` is missing.
https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/eitems/schemas/eitems/eitem-v1.0.0.json#L85

# Document form
### licenses
License field is marked with asterisk, but it's not a required field in the scheme, remove the asterisk
https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/documents/schemas/documents/document-v1.0.0.json#L426

### languages
Missing in the form.
https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/documents/schemas/documents/document-v1.0.0.json#L414

### Imprint 
The dates field should be a `DateInput` 
https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/documents/schemas/documents/document-v1.0.0.json#L343

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Closes #247

### CODE REVIEW

**Comment by kprzerwa** [comment]
there is the small fix missing with the required fields on the Location form

**Comment by KonstantinaStoikou** [comment]
The title of the exception in the location form is fixed already https://github.com/inveniosoftware/react-invenio-app-ils/blob/60b434c0e9fb5420e59cf29deaddc3afe8529542/src/lib/pages/backoffice/Location/LocationForms/LocationEditor/components/LocationForm/Closures/ExceptionsField.js#L37 and about the `is_open` toggle button it has default value `False` (https://github.com/inveniosoftware/invenio-app-ils/blob/5da6cc99934c4ff23b71d1e3b29777ab6e2c52b2/invenio_app_ils/locations/schemas/locations/location-v1.0.0.json#L126) so there is no need for the user to change it if they don't want to.

**Review by kprzerwa — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

File: src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/Imprint.js [hunk 1]
```diff
@@ -17,7 +18,7 @@ export class Imprint extends Component {
                 fieldPath="imprint.publisher"
                 optimized
               />
-              <StringField
+              <DateInputField
```
Comment (ntarocco) [inline src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/Imprint.js:21]: are we sure about this change? Backend is expecting a string, and we might want to only input a year, not a full date, same below.

Comment (kprzerwa) [reply src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/Imprint.js:21]: datepicker returns the string, so the type should be OK
but good point about the year only, @KonstantinaStoikou can you remove this change?

**Review by ntarocco — APPROVED** [review]

**Pull request merged**